frontend/src/Pages/SectionManagementForm.tsx

```diff
@@ -0,0 +1,162 @@
+import { useNavigate, useParams } from 'react-router-dom';
+import {
+    DropdownInput,
+    NumberInput,
+    SubmitButton,
+    TextAreaInput,
+    TextInput,
+    DurationInput,
+    DateInput,
+    ResetButton
+} from '@/Components/inputs';
+import { PrgSectionStatus } from '@/common';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useState } from 'react';
+import API from '@/api/api';
+
+interface SectionInputs {
+    capacity: number;
+    name: string;
+    instructor_name: string;
+    description: string;
+    duration: string;
+    duration_unit: string;
+    start_dt: Date;
+    section_status: string;
+    credit_hours: number;
+}
+
+export default function SectionManagementForm() {
+    const { id } = useParams<{ id: string }>();
+    const navigate = useNavigate();
+    const [errorMessage, setErrorMessage] = useState('');
+    const {
+        register,
+        handleSubmit,
+        reset,
+        formState: { errors }
+    } = useForm<SectionInputs>();
+
+    const onSubmit: SubmitHandler<SectionInputs> = async (data) => {
+        setErrorMessage('');
+        const formattedJson = {
+            ...data,
+            duration: data.duration + data.duration_unit,
+            start_dt: new Date(data.start_dt),
+            capacity: Number(data.capacity),
+            credit_hours: Number(data.credit_hours)
+        };
+        const response = await API.post(
+            `programs/${id}/sections`,
+            formattedJson
+        );
+        if (!response.success) {
+            //should we put a toast message here??
```

Comment:
Yes, we have a Toast Message getting displayed after creating a new program and navigating the user to the program overview page.